### ISSUE TITLE
[WEB] Undefined name 'AnotherFlushbarPlugin' in generated_plugin_registrant.dart

### ISSUE THREAD

**Issue by Ciock** [cmdrootaccess/another-flushbar#6]
This bug is relative to 1.10.9 and beyond. 
Downgrading to 1.10.8 resolve this issue.

Error: 
lib/generated_plugin_registrant.dart:21:3: Error: Getter not found: 'AnotherFlushbarPlugin'.
  AnotherFlushbarPlugin.registerWith(registrar);
  ^^^^^^^^^^^^^^^^^^^^^

```
//
// Generated file. Do not edit.
//

import 'package:another_flushbar/flushbar.dart';
import 'package:contact_picker_web/contact_picker_web.dart';
import 'package:firebase_analytics_web/firebase_analytics_web.dart';
import 'package:firebase_auth_web/firebase_auth_web.dart';
import 'package:firebase_core_web/firebase_core_web.dart';
import 'package:flutter_device_locale/src/web.dart';
import 'package:google_sign_in_web/google_sign_in_web.dart';
import 'package:location_web/location_web.dart';
import 'package:package_info_plus_web/package_info_plus_web.dart';
import 'package:shared_preferences_web/shared_preferences_web.dart';
import 'package:url_launcher_web/url_launcher_web.dart';

import 'package:flutter_web_plugins/flutter_web_plugins.dart';

// ignore: public_member_api_docs
void registerPlugins(Registrar registrar) {
  AnotherFlushbarPlugin.registerWith(registrar);
  FlutterContactPickerPlugin.registerWith(registrar);
  FirebaseAnalyticsWeb.registerWith(registrar);
  FirebaseAuthWeb.registerWith(registrar);
  FirebaseCoreWeb.registerWith(registrar);
  FlutterDeviceLocaleWebPlugin.registerWith(registrar);
  GoogleSignInPlugin.registerWith(registrar);
  LocationWebPlugin.registerWith(registrar);
  PackageInfoPlugin.registerWith(registrar);
  SharedPreferencesPlugin.registerWith(registrar);
  UrlLauncherPlugin.registerWith(registrar);
  registrar.registerMessageHandler();
}
```

**Comment by cmdrootaccess** [comment]
thanks for bringing this up. i will resolve now

**Comment by cmdrootaccess** [comment]
fixed in version 1.10.11

**Comment by nohli** [comment]
@cmdrootaccess this came back in 1.10.12
